src/extensions/CanvasWidget/CanvasPainter.ts

```diff
@@ -162,6 +162,14 @@ export class CanvasPainter {
     exportingFigure() {
         return this._exportingFigure
     }
+    // This is basically just for debugging purposes.
+    fillWholeCanvas(color: Color): void {
+        console.log(`Pixel dimensions: 0 to ${this._pixelWidth} and 0 to ${this._pixelHeight}`)
+        this._context2D.save()
+        applyBrush(this._context2D, {color: color})
+        this._context2D.fillRect(0, 0, this._pixelWidth, this._pixelHeight)
+        this._context2D.restore()
+    }
```

Comment:
This method added to `CanvasPainter` to facilitate visualizing the entire drawing field, in case we're wondering about layout issues.